index.md

```diff
@@ -72,6 +72,7 @@ benefit from these resources. You can find posts and discussion on
 - [Chinese: missing-semester-cn.github.io](https://missing-semester-cn.github.io/)
 - [Korean: missing-semester-kr.github.io](https://missing-semester-kr.github.io/)
 - [Turkish: missing-semester-tr.github.io](https://missing-semester-tr.github.io/)
+- [Traditional Chinese: missing-semester-zh-hant.github.io](https://missing-semester-zh-hant.github.io/)
```

Comment:
Can we put this one right after "Chinese" in the list? Also, I think it would look nicer if the two were named "Chinese (Simplified)" and "Chinese (Traditional)".

Reply:
Done :3